src/main/java/com/example/user/orders/controller/OrderController.java

```diff
@@ -0,0 +1,43 @@
+package com.example.user.orders.controller;
+
+import com.example.user.orders.model.request.CreateOrderRequest;
+import com.example.user.orders.model.response.OrderResponse;
+import com.example.user.orders.service.abstraction.OrderService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+
+@RestController
+@RequestMapping("v1/orders")
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+
+    @GetMapping("/{orderId}")
+    public OrderResponse getOrder(@PathVariable Long orderId){
+        return orderService.getOrder(orderId);
+    }
+
+    @GetMapping
+    public List<OrderResponse> getOrders(){
+        return orderService.getOrders();
+    }
+
+    @PutMapping("/{orderId}")
+    @ResponseStatus(NO_CONTENT)
+    public void updateOrder(@PathVariable Long orderId,
+                            CreateOrderRequest request){
```

Comment:
1. add @RequestBody for CreateOrderRequest
2. rename CreateOrderRequest to UpdateOrderRequest
3. add createOrder endpoint